mattermost/DEBIAN/postinst

```diff
@@ -1,7 +1,36 @@
 #!/usr/bin/env bash
 
+set -e
+IS_UPGRADE=false
+
 case "$1" in
     configure)
+
+    [ -z "$MATTERMOST_USER" ] && MATTERMOST_USER="mattermost"
```

Comment:
I think you are trying to do this? 
`MATTERMOST_USER=${MATTERMOST_USER:-mattermost}`

same for group

Reply:
We are trying to perform the following: 
If `MATTERMOST_USER` is null (not set) we set ot to `mattermost` 
Same for `MATTERMOST_GROUP`

Reply:
yeah this is my intent too with what I shared. you can give a try

Reply:
Tested it, seems to work as expected. 

What do we gain with the proposal? 
Wouldn't you agree that it is a more readable form? :
`[ -z "$MATTERMOST_USER" ] && MATTERMOST_USER="mattermost"` 
vs 
`MATTERMOST_USER=${MATTERMOST_USER:-mattermost}` 

The latter requires some bash insights.  

Would vote in favor of readability against saving a couple of chars. 

Let me know what do you think

Reply:
I like the second opinion but 0/5 not strong opinion. minor thing to change

Reply:
Alos prefer the 2nd one, but also 0/5